tests/unit/horizontal-bar-test.js

```diff
@@ -138,7 +138,7 @@ test("Labels aren't trimmed when width is small", function(assert) {
   this.render();
   // Charts set their defaultOuterWidth when they're rendered, so it has to be
   // set after rendering
-  Ember.run(() => component.set('defaultOuterWidth', 300));
+  Ember.run(() => { component.set('defaultOuterWidth', 300); });
```

Comment:
that does not seem related, does it?

Reply:
Same reason as https://github.com/Addepar/ember-widgets/pull/331#discussion_r425418037